modules/navier_stokes/include/postprocessors/ADCFLTimeStepSize.h

```diff
@@ -0,0 +1,51 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "ElementPostprocessor.h"
+
+/**
+ * Computes a time step size based on user-specified CFL number
+ */
+class ADCFLTimeStepSize : public ElementPostprocessor
```

Comment:
Why AD?

Reply:
I will rename to remove the AD

Reply:
Oh I see there is a `CFLTimeStepSize` object already in THM